Ilya_Torch_itorch2001@gmail.com

IlyaTorch opened the pull request:
Iteration 1
-h, --version, --json, --limit x point are ready

HenadziStantchik committed:
Add instructions in README.Add basic .py modules

IlyaTorch committed:
Iteration 1
-h, --version, --json, --limit x point are ready

dzhigailo commented on final_task/rss_parser/rss_parser.py:
```diff
@@ -0,0 +1,126 @@
+version = 1.0
+
+
+def gen_entries(parsed):
```
Here and below:
It's a good practice to add type hits in your function signatures (e.g. def gen_entires(parsed: <some  argument type>) -> <some_type>

dzhigailo commented on final_task/rss_parser/rss_parser.py:
```diff
@@ -0,0 +1,126 @@
+        entry.summary = entry.summary[:entry.summary.find('<')] + entry.summary[entry.summary.find('>') + 1:]
+
+
+def convert_to_dict(parsed, entry):
```
Take a look in `dataclasses` lib

IlyaTorch committed:
Iteration 1

add class structure for parsing console arguments and for article from
web- site

IlyaTorch committed:
Iteration 1
Combine functions for parsing console arguments, for handling options,
for handling entries into classes using dataclasses module. Add type of accepted and returned
values of methonds.
